core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AppSparkMetricsAnalyzer.scala

```diff
@@ -434,6 +518,23 @@ object AppSparkMetricsAnalyzer  {
     }
   }
 
+  /**
+   * Given an input iterable, returns its min, median, max and sum.
+   */
+  def getStatistics(arr: Iterable[Long]): (Long, Long, Long, Long) = {
```

Comment:
I updated the implementation to reuse/pull existing metrics results from `ProfStageMetricView`. I cannot do this for `shuffle read total bytes` because in `ProfStageMetricView` there are 2 metrics associated with this: `internal.metrics.shuffle.read.localBytesRead` and `internal.metrics.shuffle.read.remoteBytesRead`. I cannot get the min/med/max of `shuffle read total bytes` by adding the min/med/max of the 2 metrics. I am keeping this function for now, but if it looks too unnecessary I can remove it.